test/python/common.py

```diff
@@ -47,8 +47,10 @@ def import_pbtxt(self, pb_filename):
             tf.import_graph_def(graph_def)
         return graph
 
-    def with_ngraph(self, l, config=tf.ConfigProto()):
+    def with_ngraph(self, l, config=None):
```

Comment:
mutable default values in python function caused `update_config` to be called on the same config object multiple times, causing the i'th test to run ngraph optimizer i times